CHANGELOG

```diff
@@ -1,9 +1,13 @@
-snowplow-unified 0.1.3 (2024-01-03)
+snowplow-unified 0.2.0 (2024-0X-XX)
 ---------------------------------------
 ## Summary
 
+## Features
+- Adds user stitching to the users table (enabled with `snowplow__session_stitching`)
+
 ## Fixes
 - Consider screen view ID from the screen view context (#14)
+- Fix link to incorrect FAQ in README
```

Comment:
Previous fix needs to be removed

Reply:
That's not yet released is it?

Reply:
Oh, it looked like it did, ignore it pls.